backend/app/rest/api/admin_test.go

```diff
@@ -348,7 +348,7 @@ func TestAdmin_Block(t *testing.T) {
 	assert.Equal(t, "test test #1", comments.Comments[2].Text, "comment not removed and not cleared")
 	assert.False(t, comments.Comments[2].Deleted, "not deleted")
 
-	srv.pubRest.cache = cache.NewScache(cache.NewNopCache()) // TODO: with lru cache it won't be refreshed and invalidated for long
+	srv.pubRest.cache = cache.NewScache[[]byte](cache.NewNopCache[[]byte]()) // TODO: with lru cache it won't be refreshed and invalidated for long
```

Comment:
I would expect it to infer the type for `NewNopCache`

Reply:
Apparently it doesn't work:
```
Cannot infer V
Cannot use 'cache.NewScache(cache.NewNopCache[[]byte]())' (type *Scache[V]) as the type LoadingCache Type does not implement 'LoadingCache' need the method: Get(key lcw.Key, fn func() ([]byte, error)) (data []byte, err error) have the method: Get(key Key, fn func() (V, error)) (data V, err error)
```